### PR TITLE
docs: Document --spawn_strategy=standalone for test

### LINKED ISSUE #2571: Document --spawn_strategy=standalone for test
### Description of the problem / feature request / question:

`--spawn_strategy=standalone` actually disables sandboxing for tests. This is IMO not clear since the option  is documented under _BuildOptions_. Make it clear that it also applies for testing.

### PR DESCRIPTION
**Pull request by harmank**
Resolves #2571

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java [hunk 1]
```diff
@@ -43,7 +43,7 @@
       effectTags = {OptionEffectTag.UNKNOWN},
       help =
           "Specify how spawn actions are executed by default."
-              + "'standalone' means run all of them locally."
+              + "'standalone' means run all of them locally and it disables sandboxing for tests."
```
Comment (philwo) [inline src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java]: Not only for tests, I would change this to:

'standalone' means run all of them locally without any kind of sandboxing

File: src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java [hunk 2]
```diff
@@ -43,7 +43,7 @@
...
               + "'sandboxed' means run them in namespaces based sandbox (available only on Linux)"
```
Comment (philwo) [inline src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java]: This is also wrong by now, it should read:

'sandboxed' means to run them in a sandboxed environment with limited privileges (details depend on platform support)

File: src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java (hunk 1 above)
Comment (harmank) [inline src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java]: Thanks for the review. Updated.

File: src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java (hunk 2 above)
Comment (harmank) [inline src/main/java/com/google/devtools/build/lib/bazel/rules/BazelStrategyModule.java]: Thanks for the review. Updated

**Comment by laszlocsomor** [comment]
@philwo , did you import this PR?

**Comment by philwo** [comment]
Not yet, sorry. Will do it right now.

**Pull request closed without merging**